templates/horizon/config/local_settings.py

```diff
@@ -65,17 +65,17 @@ def get_pod_ip():
     import socket
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
     try:
-        s.connect(("{{ .horizonEndpointUrl }}", 80))
+        s.connect(("{{ .horizonEndpointHost }}", 80))
```

Comment:
in case of tls, is this still port 80?

Reply:
IIUC this doesn't matter, just opens a socket to $something to determine the pod IP.
I guess if there is a firewall blocking port 80 on the ingress controller it would block this though so might be best to use the correct port

Reply:
Yeah, correct. I just needed a way to determine the pods IP address so that we could add it to `ALLOWED_HOSTS`. The port should be ok, because port 80 is always open it just redirects to 443, and an open port of any kind is sufficient to pick up the IP address.

Reply:
However in this specific case port 80 is firewalled (not specified where/how though) so it may be safest to use the correct port for the endpoint

Reply:
Done in the latest patch. Maybe in a follow-up should look into replacing this with  [downward-api volume](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/) or [env var](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/) passing status.podIP instead